src/components/Home/index.js

```diff
@@ -26,6 +26,7 @@ class Home extends Component {
     super(props);
     this.state = {
       loading: false,
+      isEmptyEventsList: false,
```

Comment:
No need to use state. Check `itemList.length`

Reply:
Nah. It cant be. Because if there are no events, server will return an error 
![image](https://user-images.githubusercontent.com/17812909/71311060-50363400-244e-11ea-8116-716df84ba488.png)

Reply:
But I have found out another way like your suggestion